layouts/partials/blocks/candidate-results.html

```diff
@@ -0,0 +1,80 @@
+{{ with .block }}
+  {{ $data := index site.Data.candidates2024 }}
+
+  <script>
+    function candidates() {
+
+      let urlParams = new URL(window.location).searchParams.entries();
+      let candidateData = {{$data}}
+
+      function paramsToObject() {
+        const result = {};
+        for (const [key, value] of urlParams) {
+          result[key] = value;
+        }
+        return result;
+      }
+
+      let candidateScores = paramsToObject()
+
+      let orderedCandidates = candidateData.map(obj => ({
```

Comment:
This isn't sorted yet, so I would probably call it something like `candidateObjs` and maybe rename `candidates` to `sortedCandidates` or `rankedCandidates`.